src/main/java/duke/Command.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+/**
+ * Represents a function that will be called in the duke.Duke running loop.
+ */
+public class Command {
+    private TriConsumer func;
+    private boolean isExit;
+
+    /**
+     * @param func function that has to be called with taskList, ui, storage
+     * @param isExit true iff func is an exit function
+     */
```

Comment:
I think you need to add punctuation to all your javadoc as per the style guide :-)

Reply:
Yes.